common/app/pagepresser/InteractiveImmersiveHtmlCleaner.scala

```diff
@@ -22,17 +22,24 @@ object InteractiveImmersiveHtmlCleaner extends HtmlCleaner {
   }
 
   def clean(document: Document, convertToHttps: Boolean, now: LocalDateTime): Document = {
-    universalClean(document)
-    removeTopBannerAds(document)
+    partialUniversalClean(document)
     removeReaderRevenueCallouts(document)
     removeEmailSignup(document)
     removeByTagName(document, "noscript")
     if (convertToHttps) secureDocument(document)
     addExtraCopyToDocument(document, now)
   }
 
-  def removeTopBannerAds(document: Document): Document = {
-    removeByClass(document, "top-banner-ad-container js-top-banner")
+  def partialUniversalClean(document: Document): Document = {
```

Comment:
Is it worth just inlining these into the clean method? I'm not sure much is gained from the separation here, which is probably why coming up with a good name is difficult (`partialUniversalClean` is a bit of a contradiction!).

Reply:
Nice, will update :)